lib/native_interaction/rubix/rubix_platform_calls.dart

```diff
@@ -129,15 +91,25 @@ class RubixPlatform {
     String did = "";
```

Comment:
declaration not needed here, can directly declare and initialise inside the condition